site-docker.yml.sample

```diff
@@ -1,6 +1,9 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all
+  tasks: []
+
 - hosts: mons
   become: True
```

Comment:
Yup, do you mind sending a quick PR for that? I'm busy fixing the docker stuff :)